pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/HashJoinOperator.java

```diff
@@ -18,12 +20,13 @@
  */
 package org.apache.pinot.query.runtime.operator;
 
-import com.google.common.base.Preconditions;
+import com.clearspring.analytics.util.Preconditions;
```

Comment:
do not change this

Reply:
Good catch. I think it is an accidental change... I am not sure how it happens. updated.